api/api.yaml

```diff
@@ -218,12 +320,15 @@ paths:
       description: >-
         Set specific state/actions on device configuration resources on the traffic
         generator.
+      x-status:
+        status: deprecated
+        information: Please use `set_control_state` with `device` choice instead
```

Comment:
Needs change to reflect current proposed model i.e. 'set_control_state' with 'lacp' choice to trigger change in lacp_member_state

Reply:
resolved